### PR TITLE
 Issue 2 symbolic calculator window

### LINKED ISSUE #2: Add new symbolic calculator option
Feature 1; User Story A; Task 002
In order to implement symbolic calculation options for the Microsoft calculator, we need a new calculator option in the navigation pane.

### PR DESCRIPTION
**Pull request by Jared-Gibson**
Created the symbolic calculator option in the navigation panel.
closes #2

### CODE REVIEW

File: src/CalcViewModel/Common/NavCategory.cpp [hunk 1]
```diff
@@ -130,6 +137,17 @@ static const list<NavCategoryInitializer> s_categoryManifest = [] {
                                                                      L"2",
                                                                      SUPPORTS_ALL,
                                                                      true,
+                                                                     false },
+                                             NavCategoryInitializer{ ViewMode::Symbolic,
+                                                                     SYMBOLIC_ID,
+                                                                     L"Symbolic",
+                                                                     L"SymbolicMode",   //Note: if you want the navigation pane to have Symbolic on the tab, you need to go into the resource folder for a language and add a symbolic section there
+                                                                     L"\u222B", //This is the glyph for the integral symbol; however, it is not recognized as a glyph for whatever reason
```
Comment (theEhlien) [inline src/CalcViewModel/Common/NavCategory.cpp:145]: Looking into the project some, I believe that the glyphs used here are likely defined in a font file used by the program. There is a font file under the calculator repository here: "src\Calculator\Assets\CalcML2.ttf" as well as all of the pictures used for the navigation panel in the application. I am going to design and add an asset that follows these examples for the symbolic tab in a commit shortly.

**Review by theEhlien — COMMENTED** [review]
Overall, everything looks clean and consistent. There are a couple of minor concerns with the content in this request.

- The Doxygen comments, as per our instructions, should state why something is being implemented and not how. Other than this, comments look clean and precise in speech.
- I believe that an image can be added for the navigation pane tab representing symbolic mode, using a glyph that references specifically from the defined font within the assets of the solution. A comment was added to line 145 of "src\CalcViewModel\Common\NavCategory.cpp"

File: src/CalcViewModel/Common/NavCategory.cpp (hunk 1 above)
Comment (theEhlien) [reply src/CalcViewModel/Common/NavCategory.cpp:145]: I installed the font on my computer and verified that the glyphs are being referenced from this font file.

**Comment by Nicolo7** [comment]
Ran this on my computer and the tab is present. Great work, nothing breaks and its well commented after Josh's suggestion.

**Comment by smalls4894** [comment]
Excellent structure, well documented, no errors, code follows general structure for other calculator window construction. Well done

**Pull request merged**